web/themes/custom/nysenate_theme/nysenate_theme.theme

```diff
@@ -687,12 +687,29 @@ function nysenate_theme_preprocess_block_content__featured_legislation(&$variabl
         if ($bill->hasField('field_ol_last_status') && !$bill->get('field_ol_last_status')->isEmpty()) {
           $last_status = $allowed_values[$bill->field_ol_last_status->value];
         }
+
+        $committee_link = '';
```

Comment:
This section will be prone to breakage. 

1. `$committee_link` can be a string or array at the end.  It should be consistent.
2. Instead of attempting to build the URL manually, try loading the taxonomy term by name and let drupal provide the  URL.